packages/ui-components/src/components/state-indicator/state-indicator.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, Prop, h, Host } from "@stencil/core";
+
+@Component({
+	tag: 'kv-state-indicator',
+	styleUrl: 'state-indicator.scss',
+	shadow: true
+})
+export class KvStateIndicator {
+
```

Comment:
Please add comments to the variables/props so documentation can be properly generated.